### PR TITLE
ATO-414: Remove SUPPORT_AUTH_ORCH_SPLIT Config

### PR DESCRIPTION
**Pull request by BenjaminWCO**
## What

Removed SUPPORT_AUTH_ORCH_SPLIT Flag config.

## How to review
Code Review
## Checklist
- [x] Impact on orch and auth mutual dependencies has been checked.

### CODE REVIEW

File: shared-test/src/main/java/uk/gov/di/authentication/sharedtest/basetest/HandlerIntegrationTest.java [hunk 1]
```diff
@@ -187,13 +187,7 @@ public String getTxmaAuditQueueUrl() {
...
                     spotQueue,
                     docAppPrivateKeyJwtSigner,
-                    configurationParameters) {
-
-                @Override
-                public boolean isAuthOrchSplitEnabled() {
-                    return true;
-                }
-            };
+                    configurationParameters) {};
```
Comment (isaac-GDS) [inline shared-test/src/main/java/uk/gov/di/authentication/sharedtest/basetest/HandlerIntegrationTest.java:190]: Can this all be removed if we are just using `TEST_CONFIGURATION_SERVICE` for the the integration test ?

**Review by isaac-GDS — DISMISSED** [review]

**Comment by isaac-GDS** [comment]
Is it possible to rebase onto main instead of merging it. Revert the latest `merge into main commit`

File: orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/AccessTokenService.java [hunk 2]
```diff
@@ -1,85 +0,0 @@
-package uk.gov.di.orchestration.shared.services;
```
Comment (isaac-GDS) [inline orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/AccessTokenService.java:1]: Wanted to double check that this is correct - should orch and auth both have this service. Or does only auth components use this service ?

Comment (BenjaminWCO) [reply orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/AccessTokenService.java:1]: Yeah, it's unused. If you take a look at it in InteliJ, the only reference to any of the contractors of that class is from  within another one of it's contractors. I had to modify Auth's copy, but thought it made more sense to just delete ours.

**Review by isaac-GDS — APPROVED** [review]

**Pull request merged**